pygfx/helpers/_axes.py

```diff
@@ -13,11 +13,23 @@
 
 
 class AxesHelper(Line):
-    """An object indicating the axes directions.
+    """Generate an axis indicator.
```

Comment:
This first sentence is what shows up in the summary table of the module, so I like to write them in a way that they are coherent when you see them all aggregated there. (ref: https://pygfx--430.org.readthedocs.build/en/430/_autosummary/pygfx.geometries.html)

As I understand it, [Numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes) recommends documenting the constructor's arguments in the classes docstring (under `Parameters`). Since we do that, I think it makes sense to match the rest of the style as if we are documenting the constructor.

Reply:
> This first sentence is what shows up in the summary table of the module, so I like to write them in a way that they are coherent when you see them all aggregated there.

Fair enough for those use-cases.

> Since we do that, I think it makes sense to match the rest of the style as if we are documenting the constructor.

From my experience it's more common to describe the object. The fact that you also describe the parameters to instantiate such an object does not conflict IMO. This is an example from the same numpydoc page:

<img width="445" alt="image" src="https://user-images.githubusercontent.com/3015475/212407998-e76eb112-0cf8-47ba-b5ca-8a342c844142.png">

Reply:
Alright, then I will aim to write the docstrings from the classes/objects perspective.